src/AutosizeInput.js

```diff
@@ -152,7 +158,28 @@ const AutosizeInput = createClass({
 				}
 			</div>
 		);
-	},
-});
+	};
+}
+
+AutosizeInput.propTypes = {
```

Comment:
You can, but I like having the propTypes next to the defaultProps. Personal preference though